src/ekore/anomalous_dimensions/unpolarized/time_like/as1.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+"""The following are the unpolarized time-like leading order Altarelli-Parisi splitting kernels."""
+
+import numba as nb
+import numpy as np
+from eko import constants
+
+
+
+@nb.njit(cache=True)
+def gamma_qq(N, s1):
+    """
+    Computes the LO quark-quark anomalous dimension
+    Implements Eqn. (B.3) from hep-ph/0604160
```

Comment:
A first brief (less than one line) description should start on the same line of the quotes:
```suggestion
    """Computes the LO quark-quark anomalous dimension.
    
    Implements Eqn. (B.3) from hep-ph/0604160
```
and end with a dot.